courtlistener/process_cl.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import csv
+import logging
+import os
+import re
+import sys
+
+import pandas as pd
+
+from licensed_pile.licenses import PermissiveLicenses
+from licensed_pile.logs import configure_logging
+from licensed_pile.write import to_dolma
+
+SOURCE_NAME = "CourtListenerOpinion"
+
+csv.field_size_limit(sys.maxsize)
+
+logger = configure_logging("court-listener-opinion")
+
+
+def process_court_listener(file_path):
+    df = pd.read_csv(file_path)
+
+    # add metadata column
+    df["metadata"] = str(PermissiveLicenses.PD)
+
+    # add source column
+    df["source"] = SOURCE_NAME
```

Comment:
Lets add a `source_name` argument (which we can default to `SOURCE_NAME`) and set that instead of using a global.

Reply:
WIll do.

Reply:
```python
def process_court_listener(file_path):
    ...
    df["source"] = args.source_name
```
```python
    parser.add_argument(
        "--source_name",
        default="SOURCE_NAME",
        help="The name of the source.",
    )
```